### ISSUE TITLE
Small typo in error message

### ISSUE THREAD

**Issue by moseskarunia** [felangel/mocktail#30]
# What's written

```
Bad state: A test tried to use `any` or `captureAny` on a parameter of type `RefreshAccountSnapshotOptions`, but
registerFallbackValue was not previously called to register a fallback value for `RefreshAccountSnapshotOptions`

To fix, do:

void main() {
  setupAll(() {
    registerFallbackValue<RefreshAccountSnapshotOptions>(RefreshAccountSnapshotOptions());
  });
}
```

# What's should be written

`setUpAll` instead of `setupAll`.

# Note

Yeah I know it's just a small thing. But an inconsistency can feels somewhat troubling for a perfectionist. lol

Should I submit a PR for this?

**Comment by felangel** [comment]
Hi @moseskarunia 👋 
Thanks for bringing this up!

I quickly pulled it in as part of #29 and it's now published in 0.0.2-dev.6 🎉

**Comment by moseskarunia** [comment]
> Hi @moseskarunia 👋 
> Thanks for bringing this up!
> 
> I quickly pulled it in as part of #29 and it's now published in 0.0.2-dev.6 🎉 

Thank you felix!
